Lava and Water  .js

```diff
@@ -0,0 +1,307 @@
+3333333333333333
+3933333333333933
+3333339333333333
+3393333333933333
+3333333393333333
+3333393333393339
+3333333333333333
+3933339333333333
+3333933333333933
+3333333339333333
+3333333333339333
+3393333333333333
+3333333933333333
+3933333333333339
+3333393333393333
+3333333933333333`], 
+  [ blueteleport, bitmap`
+5555555555755555
+5575557555555555
+5555555555555555
+7555555555555555
+5555575555555755
+5555555557555555
+5575555555555555
+5555555555555555
+5555555755557575
+7555755555555555
+5555555555555555
+5555555555755555
+5555555555555557
+5555755575555755
+5755555555555555
+5555555555555555`], 
+  [ wall, bitmap`
+0000000000000000
+0000000000000000
+0000000000000000
+0000000000000000
+0000000000000000
+0000000000000000
+0000000000000000
+0000000000000000
+0000000000000000
+0000000000000000
+0000000000000000
+0000000000000000
+0000000000000000
+0000000000000000
+0000000000000000
+0000000000000000`], 
+  [ redlocker, bitmap`
+7777777777777777
+7777777777777777
+7777777777777777
+7777777777777777
+7777777777777777
+7777777777777777
+7777777777777777
+7777777777777777
+7777777777777777
+7777777777777777
+7777777777777777
+7777777777777777
+7777777777777777
+7777777777777777
+7777777777777777
+7777777777777777`], 
+  [ bluelocker, bitmap`
+9999999999999999
+9999999999999999
+9999999999999999
+9999999999999999
+9999999999999999
+9999999999999999
+9999999999999999
+9999999999999999
+9999999999999999
+9999999999999999
+9999999999999999
+9999999999999999
+9999999999999999
+9999999999999999
+9999999999999999
+9999999999999999`], 
+  
+);
+
+setSolids([wall, redlocker, bluelocker, player1, player2]);
+
+
+let level = 0;
+const levels = [
+  map`
+p....r
+wwwww.
+......
+wwwww.
+b....q`,
+  map`
+p.....q
+www.www
+..w.w..
+..w.w..
+..w.w..
+b.....r`, 
+  map`
+p.....r
+...I...
+..wqw..
+ww.w.w.
+.......
+...b...`, 
+  map`
+....pq.....
+wIwwwwwwwlw
+...........
+...........
+wlwwwwwwwIw
+...........
+....rb.....`, 
+  map`
+p.l..I......
+..l..I.....w
+..l..I....lr
+..l.qI.....w
+..l..I......
+..l..I.....w
+..l..I....Ib
+..l..I.....w
+..l..I......`,
+  map`
+...w...........w...
+.w.w.w.wwwww.w.w.w.
+.w.w.w.wr.bw.w.w.w.
+.w.w.w.w.w.w.w.w.w.
+.w.w.wI.l.I.lw.w.w.
+.w.w.w.......w.w.w.
+qw...w.......w...wp`, 
+];
+
+const currentLevel = levels[level];
+setMap(currentLevel);
+
+setPushables({
+  [ player1 ]: [bluelocker],
+  [ player2 ]: [redlocker], 
+});
+
+//player1 - movement 
+onInput("w", () => {
+  getFirst(player1).y += -1
+});
+
+onInput("s", () => {
+  getFirst(player1).y += 1
+});
+
+onInput("a", () => {
+  getFirst(player1).x += -1
+});
+
+onInput("d", () => {
+  getFirst(player1).x += 1
+});
+// End - player1 movement 
+
+
+//player2 movement 
+onInput("i", () => {
+  getFirst(player2).y += -1
+});
+
+onInput("k", () => {
+  getFirst(player2).y += 1
+});
+
+onInput("j", () => {
+  getFirst(player2).x += -1
+});
+
+onInput("l", () => {
+  getFirst(player2).x += 1
+});
+//End - player2 movement 
+
+
+var tempototal = 100;
```

Comment:
I'm glad you used my timer :) I noticed that you used because the variables are in Portuguese (my native language! I'm from Brazil!), I used it in my game Flurffy

in English it would look like this:

var total = 10;
var timer = setInterval(function(){
total--;
clearText();
addText(""+total, { y: 1 , color: [255,0,0] });
if(total <= 0){
clearTile(getFirst(player).x,getFirst(player).y);
clearInterval(timer);
addText("Game over!")
}
},1000);